Loop/View Models/AlertPermissionsViewModel.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  AlertPermissionsViewModel.swift
+//  Loop
+//
+//  Created by Anna Quinlan on 11/7/22.
+//  Copyright © 2022 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public class AlertPermissionsViewModel: ObservableObject {
```

Comment:
I pulled this out into a view model since it seemed more in line with what we do for other settings views